pages/contact.html

```diff
@@ -41,5 +41,23 @@
     </a>
 </div>
 </div>
+
+<div class="ui main text container">
+    <h1 class="ui dividing header" style="padding-top:8px">School Contact Details</h1>
+    <p>Email School: <a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
+    <p>Web: mountsioncbssecondary.ie</p>
+    <p>Telephone Number: 051 377378</p>
+    <p>Fax Number: 051-376468</p>
+    <p>School Roll No. 64930i</p>
+    <p>Address:</p>
+    <p>Mount Sion CBS Secondary School,</p>
+    <p>Barrack Street,</p>
+    <p>Waterford City.</p>
+    <h1 class="ui dividing header" style="padding-top:8px">School Location</h1>
+    <p>School Location on Google maps</p>
+</div>
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9769.151915664062!2d-7.1166274!3d52.2563154!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf8355ef4efe9f1a4!2sMount+Sion+CBS+Secondary+School!5e0!3m2!1sen!2sie!4v1523822538739" width="100%" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
```

Comment:
This is outside the container, could you move it back inside as it now stretches the full width of the screen instead of container